src/ScottPlot/Plottable/RadialGaugePlot.cs

```diff
@@ -0,0 +1,709 @@
+                DataAngular[i, 1] = AngleSwept;
+
+                if (DataRaw[i] >= 0)
+                    AngleSumPos += AngleSwept;
+                else
+                    AngleSumNeg += AngleSwept;
+            }
+
+            // Compute the initial angle for the background gauges
+            _StartingAngleBackGauges = _StartingAngleGauges;
+            _StartingAngleBackGauges += (_GaugeDirection == RadialGaugeDirection.AntiClockwise ? -1 : 1) * (float)(_AngleRange * _MinScale / (_MaxScale - _MinScale));
+        }
+
+        /// <summary>
+        /// Sets the value of <see cref="MaxScale"/> and <see cref="MinScale"/> properties
+        /// </summary>
+        private void ComputeMaxMin()
+        {
+            if (GaugeMode == RadialGaugeMode.Sequential || GaugeMode == RadialGaugeMode.SingleGauge)
+            {
+                _MaxScale = DataRaw.Sum(x => Math.Abs(x));
+                _MinScale = 0;
+            }
+            else
+            {
+                _MaxScale = DataRaw.Max(x => Math.Abs(x));
+                var min = DataRaw.Min();
+                _MinScale = min < 0 ? min : 0;
+            }
+        }
+
+        /// <summary>
+        /// Needed as part of IPlottable in ScottPlot.ScottForm
+        /// </summary>
+        /// <param name="deep"></param>
+        public void ValidateData(bool deep = false)
+        {
+            if (GaugeLabels != null && GaugeLabels.Length != DataRaw.Length)
+                throw new InvalidOperationException("Gauge labels must match size of data values");
+        }
+
+        /// <summary>
+        /// Needed as part of IPlottable in ScottPlot.ScottForm
+        /// </summary>
+        /// <returns></returns>
+        public LegendItem[] GetLegendItems()
+        {
+            if (GaugeLabels is null)
+                return null;
+
+            List<LegendItem> legendItems = new List<LegendItem>();
+            for (int i = 0; i < GaugeLabels.Length; i++)
+            {
+                var item = new LegendItem()
+                {
+                    label = GaugeLabels[i],
+                    color = GaugeColors[i],
+                    lineWidth = 10,
+                    markerShape = MarkerShape.none
+                };
+                legendItems.Add(item);
+            }
+
+            return legendItems.ToArray();
+        }
+
+        /// <summary>
+        /// Needed as part of IPlottable in ScottPlot.ScottForm
+        /// </summary>
+        /// <returns></returns>
+        public AxisLimits GetAxisLimits() =>
+            (GaugeLabels != null) ? new AxisLimits(-3.5, 3.5, -3.5, 3.5) : new AxisLimits(-2.5, 2.5, -2.5, 2.5);
+
+        /// <summary>
+        /// Reduces an angle into the range [0°-360°]
+        /// </summary>
+        /// <param name="angle">Angle value</param>
+        /// <returns>Return the angle whithin [0°-360°]</returns>
+        private double ReduceAngle(double angle)
```

Comment:
This return the wrong answer for negative numbers.

i.e. `ReduceAngle(-365) == -725` 

I think a modulus based approach would be simpler.

```cs
private double ReduceAngle(double angle)
{
    double reduced = angle % 360;
    if (angle < 0) {
        angle += 360;
    }
}
```

Reply:
I think the present code also breaks if angles are >720º, and the suggested code breaks if the angle is <-360º. I'll refactor this now to use something like @bclehmann's suggested method using a `while` instead of `if`... but it will break if the user passes-in `double.NegativeInfinity` or `double.NaN` 😆

It's probably more effort than I want to invest right now, but if there's shared rotational logic between multiple plot types it may be a good idea to eventually break this out into a static class somewhere and add several unit tests to verify it works as expected for many different values.

Reply:
... my bad, @bclehmann's code doesn't break for <-360º values, so that `while` I mentioned isn't needed

I'm still not going to work about `double.NegativeInfinity` or `double.NaN` 🙃